setup.cfg

```diff
@@ -100,7 +100,7 @@ install_requires =
     funcsigs>=1.0.0, <2.0.0
     graphviz>=0.12
     gunicorn>=19.5.0, <20.0
-    importlib_resources;python_version<"3.7"
+    importlib_resources~=1.4
```

Comment:
We still should have the python version constraint - this is in the standard library from py3.8

Reply:
This is not why PyPI description of importlib_resources says. If I read the documentation correctly, we should only constraint it once we add 3.9 support:

From: https://pypi.org/project/importlib-resources/
```
importlib_resources is a backport of Python standard library importlib.resources module for Python 2.7, and 3.4 through 3.8. Users of Python 3.9 and beyond should use the standard library module, since for these versions, importlib_resources just delegates to that module.
```

I believe importlib.resources is available as of 3.7 (not as of 3.8), but it is not yet a fully-featured replacement for importlib_resources until 3.9 and we should install it in all: 3.6, 3.7, 3.8. 

Where do you get this <3.8 limitation from?

Reply:
From memory - on mobile while having coffee so wasn't easy for me to look up exact detail.

Cool 👍🏻